test/unit/mux.spec.cjs

```diff
@@ -20,7 +20,7 @@ describe('Unit::Mux', () => {
 
     /** @test {Mux.JTW} */
     it('exposes JWT Helper utilities as static methods', () => {
-      expect(Mux.JWT.sign).to.be.a('function');
+      expect(Mux.JWT.signPlaybackId).to.be.a('function');
```

Comment:
Probably want to add a test for signSpacesId as well, just to try to be thorough